universal_computation/experiment.py

```diff
@@ -186,10 +185,12 @@ def accuracy_fn(preds, true, x=None):
             **exp_args,
             **kwargs,
         )
+        del config['__dict__']
+        del config['__weakref__']
```

Comment:
Just out of curiosity, why are you deleting such values from config?

Reply:
I am not sure :P but otherwise there were some exceptions with JSON serialization of `config` dict 🤷